docs/guides/testing-typescript-with-tsnode.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Testing TypeScript with tsnode
+custom_edit_url: https://github.com/stryker-mutator/stryker/edit/master/docs/guides/testing-typescript-with-tsnode.md
+---
+
+## Projects using `ts-node` to test TypeScript files
+
+If you are creating a project which tests TypeScript source files using `ts-node` as a just-in-time transpiler, the following setup may be required for you.
+
+If you manage the compilation of your code through a `tsconfig.json` file, this guide ensures that your TypeScript code is compiled against that configuration when it is being run in the Stryker sandbox.
+
+> This guide does not cover using `@stryker-mutator/typescript-checker`. Please review the page [here](../typescript-checker.md) if you would like to include the typescript checker.
+
+### Configuration
+
+The following _additional_ items may be required to add to your `stryker.conf.js` file. This guide assumes that you have already generated a Stryker configuration file, so the following properties will need either adding or changing within the existing configuration.
+
+```js
+/**
+ * @type {import('@stryker-mutator/api/core').StrykerOptions}
+ */
+module.exports = {
+  buildCommand: 'tsc --build path/to/your/tsconfig.json',
+  commandRunner: {
+    command: 'npm run test:ts',
```

Comment:
I would prefer to have a guide using the mocha test runner. Reasons are:

1. A nodejs project will probably use either jest or mocha. Jest does its own transpiling, so makes sense to focus on mocha.
2. Using a test runner plugin will dramatically enhance performance because they can use coverage analysis.

Leaving it like this would mean to create a `"scripts": { "test:ts": "..." }` script in your package.json file and this guide doesn't specify that part currently. You can look at [mocha-ts-node](https://github.com/stryker-mutator/stryker/tree/master/e2e/test/mocha-ts-node) e2e test for an example.

Maybe it makes sense to create 2 chapters in this guide? One per test runner (jest vs mocha). You might want to mention that jest doesn't need the use of ts-node at all.

Reply:
Second attempt at explaining the configuration process, now with specific references to mocha and jest.